README.md

```diff
@@ -266,15 +245,8 @@ spacing, especially after running prettier.
 
 ## Internationalization
 
-| ❗ Currently the Wiki is being reorgnaized and updated. It is recommended to resume working on translations after the Wiki revamp is completed. |
-| ----------------------------------------------------------------------------------------------------------------------------------------------- |
-
-We are using Crowdin to manage all different translations. You can go to the
```

Comment:
We can restore this if required but given that it is currently inactive I removed the details.

I was also wondering if it makes sense to remove everything under https://github.com/w3f/polkadot-wiki/tree/master/polkadot-wiki/translated_docs?  If this information is out-of-date or incorrect does it make sense to keep it?  We can always recover it through the git history.  Removing these files also removes the requirement of having to use the `files to excludes: polkadot-wiki/translated_docs` filter in vs code when searching.

Reply:
I agree

Reply:
Awesome, I will follow up with another PR that just removes that directory :+1: